.travis.yml

```diff
@@ -4,8 +4,6 @@ python:
 
 cache:
   directories:
-    - schema_editor/node_modules
```

Comment:
That's certainly annoying but this seems like the only way to address it. What's the effect of not caching here?